Scripts/ExtractDomainFromUrlFormat/ExtractDomainFromUrlFormat.py

```diff
@@ -6,6 +6,9 @@
 from urllib import unquote
 import re
 
+# This script is highly similar to 'ExtractFQDNFromUrlAndEmail', they are not unified due to run time performance.
```

Comment:
this should be bolder.
in the current state is could be easily missed.
add a frame to the message using `==========`